spec/reek/smells/attribute_spec.rb

```diff
@@ -3,13 +3,10 @@
 require 'reek/core/module_context'
 require 'reek/smells/smell_detector_shared'
 
-include Reek::Core
-include Reek::Smells
-
-describe Attribute do
+describe Reek::Smells::Attribute do
   before :each do
-    @source_name = 'ticker'
-    @detector = Attribute.new(@source_name)
+    @source_name = 'dummy_source'
+    @detector = build(:smell_detector, smell_type: :Attribute, source: @source_name)
```

Comment:
I'm not sure this is simpler/clearer than `Reek::Smells::Attribute.new(source: @source_name)`.